### ISSUE TITLE
Ensure correct production startup for performance

### ISSUE THREAD

**Issue by stan-dot** [DiamondLightSource/blueapi#575]

**Comment by Relm-Arrowny** [comment]
I made an attempt to use gunicorn worker than I realise what we have already is probably perfectly acceptable, we are already doing 
> or with fastapi https://fastapi.tiangolo.com/deployment/docker/
> 
> the current solution looks like the use of developer setup in production

And kubernetes should be able to do all the load balancing that it required, so I think all we really need to do here is move the import to the right place?

**Comment by stan-dot** [comment]
hmm
I looked again and it would make sense.

Should we close it @callumforrester ?

**Comment by callumforrester** [comment]
Yes, I think @keithralphs may be moving the import to the right place as part of a separate ticket.
